modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Instructions.kt

```diff
@@ -216,4 +239,20 @@ object Instructions {
             )
         )
     }
+
+    private fun transferSome(sourceId: IdBox, value: Value, destinationId: IdBox): Instruction.Transfer {
```

Comment:
Are you going to reuse this method by other instruction? AFAIK only `Transfer` instruction applicable here